### PR TITLE
Fix/tca 620 button link announcement

### PR DESCRIPTION
**Pull request by lecosson**
Related to : https://oat-sa.atlassian.net/browse/TCA-620
 
test cards announced twice: as link and as button
 
#### How to test
 
- prepare an instance of TAO
- prepare an any delivery 
- open deliveries list and navigate via keyboard to listen SR announces
- ensure test cards not announced as links
   
Companion PR :
 - [ ] https://github.com/oat-sa/extension-tao-delivery/pull/483

### CODE REVIEW

**Review by AnastasiaArcadia — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: views/scss/inc/_login.scss [hunk 1]
```diff
@@ -69,6 +69,8 @@ $entryArrowHeight: 150;
     .entry-point {
 
         padding-left: 40px;
+        min-height: $entryArrowHeight * 1px;
```
Comment (ampaveliev) [inline views/scss/inc/_login.scss:72]: Looks strange. Can't understand the goal of multiplying on 1

**Review by ampaveliev — APPROVED** [review]

File: views/scss/inc/_login.scss (hunk 1 above)
Comment (lecosson) [reply views/scss/inc/_login.scss:72]: It's just moved from another style definition, not added, so think it's ok

Comment (ampaveliev) [reply views/scss/inc/_login.scss:72]: And it won't work without it?

**Pull request merged**